src/plugins/router/index.ts

```diff
@@ -204,4 +204,13 @@ const router = createRouter({
   },
 });
 
+router.onError((error, to) => {
+  if (error.message.includes('Failed to fetch dynamically imported module')) {
+    console.log('ERROR IN ROUTE; ', error.message);
```

Comment:
Maybe we should capture this in Sentry so we know when it's happening?

Reply:
Good idea. Done.